catalog/dags/popularity/refresh_popularity_metrics_task_factory.py

```diff
@@ -68,33 +67,22 @@ def create_refresh_popularity_metrics_task_group(
             },
         )
 
-        drop_constants = PythonOperator(
-            task_id=DROP_MEDIA_POPULARITY_CONSTANTS_TASK_ID,
-            python_callable=sql.drop_media_popularity_constants,
-            op_kwargs={
-                "postgres_conn_id": POSTGRES_CONN_ID,
-                "media_type": media_type,
-            },
-            execution_timeout=execution_timeout,
-            doc="Drops the popularity constants view.",
-        )
-
-        create_constants = PythonOperator(
-            task_id=CREATE_MEDIA_POPULARITY_CONSTANTS_TASK_ID,
-            python_callable=sql.create_media_popularity_constants_view,
+        update_constants = PythonOperator(
+            task_id=UPDATE_MEDIA_POPULARITY_CONSTANTS_TASK_ID,
+            python_callable=sql.update_media_popularity_constants,
             op_kwargs={
                 "postgres_conn_id": POSTGRES_CONN_ID,
                 "media_type": media_type,
             },
             execution_timeout=execution_timeout,
             doc=(
-                "Recreates the popularity constants view. This completely "
+                "Updates the popularity constants view. This completely "
```

Comment:
Is this comment accurate? Does it _completely_ recalculate the constants?

Reply:
It does, yes. It involves calculating the 80th percentile popularity score for each provider, from which new, updated constants are derived. This is done periodically as new data is ingested for providers.